extensions/wikia/JSVariables/JSVariables.php

```diff
@@ -78,7 +78,12 @@ function wfJSVariablesTopScripts(Array &$vars, &$scripts) {
 
 	$vars['wgTransactionContext'] = Transaction::getAttributes();
 
-	$scripts .= Html::inlineScript("var wgNow = new Date();") .	"\n";
+	$scripts .= Html::inlineScript("var wgNow = new Date();") . "\n";
+
+	// ADEN-6676: Inline InstantGlobals to prevent AdBlock block
+	$instantGlobalsModule = new InstantGlobalsModule();
+	$instantGlobals = $instantGlobalsModule->getScript(new ResourceLoaderContext(new ResourceLoader(), $wg->Request));
+	$scripts .= Html::inlineScript($instantGlobals) . "\n";
```

Comment:
You can minify js before output: `\JavaScriptMinifier::minify( $instantGlobals )`

BTW. You're missing a lot of spaces: https://www.mediawiki.org/wiki/Manual:Coding_conventions/PHP

Reply:
Thanks, minified. And these coding conventions are straight from hell, but reformatted.